mac/libusb/libusb.h

```diff
@@ -25,6 +25,8 @@
 #ifndef LIBUSB_H
 #define LIBUSB_H
 
+#include "mac/libusb/Xcode/config.h"
```

Comment:
This should be unnecessary, libusb.pri should take care of it.  Is there an issue with cmake or xcode?  In any event let's keep any needed changes in a separate commit and PR.

Reply:
cmake works for me:
```
-- Defines are: "HAVE_UNISTD_H;HAVE_STDARG_H;HAVE_LIBUSB_1_0;MAXIMAL_ENABLED;FILTERS_ENABLED;SHAPELIB_ENABLED;CSVFMTS_ENABLED"
-- Libs are: "Qt5::Core"
-- Include Directores are: "/Users/administrator/serialnames2.1/zlib;/Users/administrator/serialnames2.1/mac/libusb;/Users/administrator/serialnames2.1/mac/libusb/Xcode"
-- Configuring done
-- Generating done
```

Reply:
This also works for me, but now I am just guessing what you might be doing
```
cmake -G Xcode .
xcodebuild -project GPSBabel.xcodeproj -configuration Release
```

"qmake -spec macx-xcode" does not work because of the configuration tests.

Reply:
I have a fix for generating xcode projects with qmake if that's an issue.